src/api/article/article.model.js

```diff
@@ -58,9 +58,11 @@ const ArticleSchema = new Schema(
 
 const Article = mongoose.model('Article', ArticleSchema);
 
-export const serializeArticle = article => {
+// includeCollection flag here is to be able to avoid including collections
+// in case we're serializing an article into the ArticleCollection object..
+export const serializeArticle = (article, includeCollection = true) => {
```

Comment:
better use object as second param: `{ includeCollection = true }`,

compare:
```js
serializeArticle(article, false);
serializeArticle(article, { includeCollection: false });
```

Reply:
That's a nice approach!